portfolio/src/main/webapp/images.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Tyler's Image Gallery</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div id="content">
+      <h1>Tyler's Image Gallery</h1>
+      <div class="image-thumbnails">
+        <a href="images/tylerpic.jpg"><img src="images/tylerpic.jpg"/></a>
```

Comment:
It might be nice to add an "alt" tag for the image, just in case users have slow connections or other issues with images loading. This is also important for accessibility because visually impaired users won't know what the image is showing.